sphinx_needs/builder.py

```diff
@@ -158,3 +157,69 @@ def build_needumls_pumls(app: Sphinx, _exception: Exception) -> None:
         needs_builder.set_environment(env)
 
     needs_builder.finish()
+
+
+class NeedsIdBuilder(Builder):
+    name = "needs_id"
```

Comment:
Please describe, what this builder does in a docstring for the class

Reply:
I dont' make sense this. I noted there to discuss in the next meeting with you

Reply:
As this is the second builder in `builder.py`, our builder-classes should have some code-related description to explain to other developers what they shall do.

Something like:
```python

class NeedsIdBuilder(Builder):
"""  Json builder for needs, which creates one single json-file per need"""
```